account_export_csv/wizard/account_export_csv.py

```diff
@@ -245,13 +242,14 @@ def action_manual_export_journal_entries(self):
                 file_data.seek(0)
                 base64.encode(file_data, base64_data)
                 base64_data.seek(0)
-                self.env.cr.execute(
-                    """
-                UPDATE account_csv_export
-                SET data = %s
-                WHERE id = %s""",
-                    (base64_data.read(), self.id),
-                )
+                self.write({"data": base64_data.read()})
```

Comment:
Did you check the docstring of this method about the memory usage? If you change it to ORM, maybe you can provide some benchmarks to show that its memory usage is no longer of a different order than using SQL?